akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ClientSpec.scala

```diff
@@ -158,6 +162,20 @@ class Http2ClientSpec extends AkkaSpecWithMaterializer("""
             .withEntity(Strict(ContentTypes.NoContentType, ByteString.empty))
         )
       }
+
+    }
+
+    "send settings" should {
+      abstract class SettingsSetup extends TestSetupWithoutHandshake with NetProbes with Http2FrameSending {
+        def expectSetting(expected: Setting): Unit = {
+          toNet.expectBytes(Http2Protocol.ClientConnectionPreface)
```

Comment:
I would be OK with having those directly in the test rather than introducing `SettingsSetup`, but :+1: